.pre-commit-hooks.yaml

```diff
@@ -43,7 +43,7 @@
 - id: flake8-lint
   name: flake8-lint
   description: '`flake8` is a command-line utility for enforcing style consistency across Python projects.'
-  entry: 'flake8 --ignore=E501,W503'
+  entry: 'flake8 --ignore=E501,W503,T001,T002,T003,T004'
```

Comment:
This would make it not work for PR's correct? 

Check my fix out https://github.com/Flux7Labs/central-repo-git-hooks/pull/24